### DISCUSSION TITLE
Installing On Linux Mint

### DISCUSSION THREAD

**Discussion by dfghdfghfgjh** [Anjok07/ultimatevocalremovergui#959] [Q&A]
Hello, I can't seen to manage installing this on Linux. I've check other threads and it doesn't work for me. Help would be greatly appreciated!

**Comment by Pedulla** [comment]
[https://github.com/Anjok07/ultimatevocalremovergui#linux-installation](url)
This works on Linux Mint 21.x.

But, you have to edit the "requirements.txt" file and remove the line "Dora==0.0.3"
See solution here: [https://github.com/Anjok07/ultimatevocalremovergui/issues/919#issuecomment-1844123180](url)

**Reply by ghost** [reply]
Hi, the links are wrong or? 

For Linux Mint this program is to be compiled: no .deb or similar. The webpage says:


Linux Installation
See Linux Installation Instructions

These install instructions are for Debian & Arch based Linux systems.

    Download and save this repository [here](https://github.com/Anjok07/ultimatevocalremovergui/archive/refs/heads/master.zip)
    From the saved directory run the following commands in this order-

For Debian Based (Ubuntu, Mint, etc.):

sudo apt update && sudo apt upgrade
sudo apt-get update
sudo apt install ffmpeg
sudo apt install python3-pip
sudo apt-get -y install python3-tk
pip3 install -r requirements.txt
python3 UVR.py

For Arch Based (EndeavourOS):

sudo pacman -Syu
sudo pacman -Sy
sudo pacman -S python-pip
sudo pacman -S --noconfirm tk
sudo pacman -S ffmpeg

To bypass environment setup and proceed with the installation, use:

    Take caution; this modifies system files.

sudo rm /usr/lib/python3.11/EXTERNALLY-MANAGED

Then proceed with the following in order:

chmod +x install_packages.sh
./install_packages.sh
python UVR.py

Other Application Notes

    Nvidia RTX 1060 6GB is the minimum requirement for GPU conversions.
    Nvidia GPUs with at least 8GBs of V-RAM are recommended.
    AMD Radeon GPU supported is limited at this time.
        There is currently a working branch for AMD GPU users [here](https://github.com/Anjok07/ultimatevocalremovergui/tree/v5.6-amd-gpu)
    This application is only compatible with 64-bit platforms.
    This application relies on the Rubber Band library for the Time-Stretch and Pitch-Shift options.
    This application relies on FFmpeg to process non-wav audio files.
    The application will automatically remember your settings when closed.
    Conversion times will significantly depend on your hardware.
    These models are computationally intensive.

Performance:

    Model load times are faster.
    Importing/exporting audio files is faster.
